src/views/home/components/CopyEditionModal.vue

```diff
@@ -1,6 +1,7 @@
 <template>
     <b-modal
         v-if="currentEdition"
+        :visible="currentEdition !== null"
```

Comment:
If currentEdition is null, the v-if is going to fail. When is :visible false and v-if true?

Reply:
Alltough it sounds illogical, we must add: :visible="currentEdition !== null" , with this format exactly. 
Only adding this line solves the problem of the copy window not showing on the 1st click.
Also typing differently :visible="currentEdition" results with this exception:
vue.runtime.esm.js?2b0e:619 [Vue warn]: Invalid prop: type check failed for prop "visible". Expected Boolean, got Object